src/psyclone/tests/dynamo0p3_transformations_test.py

```diff
@@ -61,6 +61,31 @@
                          "test_files", "dynamo0p3")
 
 
+def test_vector_halo_exchange_remove():
+    '''test that we remove vector halo exchanges when they are no longer
```

Comment:
Please add capitalisation and punctuation.

Reply:
Oh, and maybe "halo exchanges for all components of a vector field when they..."

Reply:
Better thanks.